packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1519,6 +1519,7 @@ export const Dropdown: ComponentWithAs<'div', DropdownProps> &
 
             // Replicating same config that Downshift uses
             const actions = computeScrollIntoView(nodeToScroll, {
+              boundary: menu,
```

Comment:
Perhaps this might be a good addition ? 

```suggestion
              boundary: menu, // Explicitly set boundary to avoid unnecessary scrolling by checking all parent elements
```